### DISCUSSION TITLE
Signal store single entity signal

### DISCUSSION THREAD

**Discussion by jamestiberius** [ngrx/platform#4634] [Q&A]
Hello, using withEntities() in a signal store, I would like to obtain a Signal<Book> for each of the entities present and get the related update signals, always "x entity", by providing the respective id, as with a traditional selector. How can I achieve this? Even using a parametric rxMethod, I always end up with the entity instance but not its signal. I think I am approaching this incorrectly...
     
```
  withMethods(...
      getBookA(id: string) {
        return store.booksEntityMap()[id]
      },
      getBookB: rxMethod<string>(pipe(
        map(id => store.booksEntityMap()[id]),
      )),
      getBookC(id: string) {
        return computed(() => <Book>store.booksEntityMap()[id])
      },
```

**Comment by muuvmuuv** [comment]
Searching for the exact same, what would be the general approach here?
